CRITICAL - 2020-11-02 04:37:55 --> Unable to connect to the database.
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Apache24\htdocs\rosas\app\Models\Querys.php(21): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_usuarios('select', Array)
#4 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(45): authenticated()
#5 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#6 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-02 04:38:29 --> Unknown column 'id_usuario' in 'where clause'
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(21): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_usuarios('select', Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(45): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 04:41:22 --> Unknown column 'id_usuario' in 'where clause'
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(21): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_usuarios('select', Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(45): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 04:41:54 --> Unknown column 'idusuario' in 'where clause'
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(21): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_usuarios('select', Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(45): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 04:44:58 --> Table 'sql10373828.rosas_view_usuarios' doesn't exist
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Apache24\htdocs\rosas\app\Models\Querys.php(21): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\Apache24\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_usuarios('select', Array)
#6 C:\Apache24\htdocs\rosas\app\Controllers\BaseController.php(45): authenticated()
#7 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 04:46:07 --> Call to undefined function App\Controllers\authenticated()
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-02 04:47:14 --> Call to undefined function App\Controllers\authenticated()
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-02 05:37:46 --> Undefined property: App\Controllers\Auth::$db
#0 C:\Apache24\htdocs\rosas\app\Controllers\Auth.php(40): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\Apache24\\htd...', 40, Array)
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->authenticate()
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-02 05:38:29 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::getResultArray()
#0 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->authenticate()
#1 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 C:\Apache24\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\Apache24\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-02 20:27:45 --> Unknown column 'idusuario' in 'where clause'
#0 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\xampp\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\xampp\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 20:27:46 --> Unknown column 'idusuario' in 'where clause'
#0 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\xampp\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\xampp\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 20:27:48 --> Unknown column 'idusuario' in 'where clause'
#0 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\xampp\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\xampp\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 20:27:55 --> Unknown column 'idusuario' in 'where clause'
#0 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\xampp\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\xampp\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 20:28:23 --> Unknown column 'idusuario' in 'where clause'
#0 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\xampp\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\xampp\htdocs\rosas\app\Controllers\Auth.php(27): authenticated()
#7 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->login()
#8 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 20:28:25 --> Unknown column 'idusuario' in 'where clause'
#0 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\xampp\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\xampp\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 21:16:32 --> Table 'dbrosas.rosas_view_users' doesn't exist
#0 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\xampp\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\xampp\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-11-02 21:16:42 --> Table 'dbrosas.rosas_view_users' doesn't exist
#0 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\rosas\app\Models\Querys.php(18): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#5 C:\xampp\htdocs\rosas\app\Helpers\Rosas_helper.php(8): App\Models\Querys->view_users(Array)
#6 C:\xampp\htdocs\rosas\app\Controllers\BaseController.php(48): authenticated()
#7 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#8 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#9 C:\xampp\htdocs\rosas\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\rosas\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
